docs/mp/grpc/server-side-services.adoc

```diff
@@ -44,7 +52,22 @@ include::{rootdir}/includes/dependencies.adoc[]
 </dependency>
 ----
 
-== Defining a Service
+
+== API
+The following annotations are used to implement Helidon MP gRPC Services:
+
+* `@Grpc` - an annotation used to mark a class as representing a gRPC service.
+
+gRPC types of method:
```

Comment:
gRPC method types:

Reply:
Already updated in in PR https://github.com/oracle/helidon/pull/4606